experiments/standalone/Soil/richards_runoff.jl

```diff
@@ -72,10 +178,19 @@ soil_params = ClimaLand.Soil.RichardsParameters(;
     θ_r = θ_r,
 )
 
-function precip_function(t)
-    return -1e-6
-end
-precip = TimeVaryingInput(precip_function)
+era5_artifact_path = "/groups/esm/ClimaArtifacts/artifacts/era5_land_forcing_data2021"# @clima_artifact("era5_land_forcing_data2021")
+ref_time = DateTime(2021);
+t_start = 0.0
+# Precipitation:
+precip = TimeVaryingInput(
+    joinpath(era5_artifact_path, "era5_2021_0.9x1.25.nc"),
+    "tp",
+    surface_space;
+    reference_date = ref_time,
+    t_start,
+    regridder_type = :InterpolationsRegridder,
```

Comment:
Throughout this PR, you could set `regridder_type` somewhere and pass `(; regridder_type)`. This way, if one wants to change the regridder type, they would only have to change one place